pilot/docker/Dockerfile.proxyv2

```diff
@@ -1,4 +1,4 @@
-FROM istionightly/base_debug
+FROM gcr.io/distroless/cc
```

Comment:
/hold - please instead change base_debug to be build from distroless, and have all istio images be based on the base.

Reply:
@costinm that won't work.  If you look deeper in the PRs, you will see that several files are copied in (such as the ca certs) to our existing images during the build process and untarred.

I also think there have been some communication problems here.  This was specifically an experiment I asked to be done to test - across Istio - if the gates could be made to work with distroless.

My thinking was not to replace base as you propose, but instead, have a second set of docker files specifically tuned for distroless/small attack surface for the purposes of security and compliance.

My original experiment request was here:
https://github.com/istio/istio/pull/12694#issuecomment-476124671

I think since the experiment seems to work (well enough) - the next step is to sort out how/if we  want to introduce a second set of distroless tuned dockerfiles into the repository.